operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -162,4 +162,12 @@ data:
       - record: cluster:kube_pod_container_resource_requests:memory:sum
         expr: sum(sum(sum(kube_pod_container_resource_requests{resource=\"memory\"}) by (pod,namespace,container) * on(pod,namespace) group_left(phase) max(kube_pod_status_phase{phase=~\"Running|Pending|Unknown\"} >0) by (pod,namespace,phase)) by (pod,namespace,phase))
       - record: sli:apiserver_request_duration_seconds:trend:1m
-        expr: sum(increase(apiserver_request_duration_seconds_bucket{apiserver=\"kube-apiserver\",le=\"1\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m])) / sum(increase(apiserver_request_duration_seconds_count{apiserver=\"kube-apiserver\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m]))
\ No newline at end of file
+        expr: sum(increase(apiserver_request_duration_seconds_bucket{apiserver=\"kube-apiserver\",le=\"1\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m])) / sum(increase(apiserver_request_duration_seconds_count{apiserver=\"kube-apiserver\",verb=~\"POST|PUT|DELETE|PATCH\"}[1m]))
+      - record: cluster:policy_governance_info:propagated_count
```

Comment:
is metrics policy_governance_info available in all clusters or only in hub?
how many time series data for metrics policy_governance_info{type="propagated"} for one cluster?

if this metrics does not lead to much time series data, it's not recommended to use recording rule becuase each recording rule will add one addtional query call in managed cluster side for each scrape loop

Reply:
@JustinKuli 
if the metrics only in hub side, it's not necessary to use recording rule to handle it. You can just simply include policy_governance_info in allowlist directly.